scripts/packet.js

```diff
@@ -265,24 +267,26 @@ define(['./data-factory'], function (dataFactory) {
         return data;
     };
 
-    createStartDataPacket = function (size) {
+    createStartDataPacket = function (transactionId, size) {
         var data = dataFactory.create();
 
         data.setDword(headerLength, transactionId);
-        data.appendDword(size);
+        data.appendDword(size); // TODO(gswirski): this should be a single qword
```

Comment:
Did you manage to work out why your added `appendQword` appended two identical 4 byte values rather than correctly appending a 8 byte Qword? I tried changing `setDword` above rather than this hack, but can't see anything wrong with your `appendQword` function!

Reply:
Didn't have time to investigate it yet.